server/service/src/invoice/prescription/update/generate.rs

```diff
@@ -85,7 +92,11 @@ fn should_update_batches_total_number_of_packs(
     }
 }
 
-fn set_new_status_datetime(invoice: &mut InvoiceRow, status: &Option<UpdatePrescriptionStatus>) {
+fn set_new_status_datetime(
+    invoice: &mut InvoiceRow,
+    status: &Option<UpdatePrescriptionStatus>,
+    prescription_datetime: Option<NaiveDateTime>,
```

Comment:
This would mean prescription datetime would need to be present in the API call when status is changed ? How would this work ?

Reply:
It's been clarified that prescription_date will default = existing_invoice.allocated_datetime. Thus prescription picked and verified datetime will always = allocated_datetime, which I think works nicely, we just have to document that at some point